language/nodejs/build-images.md

```diff
@@ -140,10 +140,10 @@ Usually the very first thing you do once you’ve downloaded a project written i
 Before we can run `npm install`, we need to get our `package.json` and `package-lock.json` files into our images. We use the `COPY` command to do this. The  `COPY` command takes two parameters. The first parameter tells Docker what file(s) you would like to copy into the image. The second parameter tells Docker where you want that file(s) to be copied to. We’ll copy the `package.json` and `package-lock.json` file into our working directory `/app`.
 
 ```dockerfile
-COPY ["package.json", "package-lock.json*", "./"]
```

Comment:
The star `*` is intentional. This combined `COPY` instruction uses just one layer and can copy the `package.json` and an optional(!) `package-lock.json`. This can happen when you create a fresh project and never ran `nom install` on the host yet.